SD-K9/src/app/components/floor-plan/floor-plan.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+
+import { FloorPlanComponent } from './floor-plan.component';
+import { HttpClientModule } from '@angular/common/http';
+
+// describe('FloorPlanComponent', () => {
```

Comment:
why are the tests commented out?

Reply:
The focus of this pr is to provide the factory pattern and the new file structure, since a lot of issues depend on this.
The tests shall be updated once this is completely integrated.

Reply:
ok, but create a task in the issue to update the tests please.